code/modules/mob/living/superior_animal/giant_spider/ai.dm

```diff
@@ -1,161 +1,2 @@
-/mob/living/superior_animal/giant_spider/proc/FindTarget()
```

Comment:
All of this has been centralised into the main superior animal life, right?

Reply:
Yes.